### DISCUSSION TITLE
List all supported platforms in the readme

### DISCUSSION THREAD

**Discussion by janosh** [spences10/sveltekit-embed#387] [Ideas]
Would be great if you could see at a glance from the readme if `sveltekit-embed` has a component for a given platform.

**Comment by spences10** [comment]
Hey @janosh yeah, I think I _had_ something like that but it turns out it's just a Tweet 😅 

https://twitter.com/spences10/status/1521444473212788897

Feel free to make a PR, I'll probably need this on the example site too

**Comment by spences10** [comment]
Added the platforms to the readme with 70287785182a4f4db8d26be4e49c30bce4ff36bb

**Reply by janosh** [reply]
Thanks!
